yb-voyager/cmd/importData.go

```diff
@@ -766,7 +768,7 @@ func getTablesToImport() ([]string, []string, []string, error) {
 		}
 	}
 
-	return doneTables, interruptedTables, remainingTables, nil
+	return doneTables, removeExcludeTables(interruptedTables), removeExcludeTables(remainingTables), nil
```

Comment:
I think you should this filtering at one place just once only.
Either in this `getTablesToImport` functions or in the `allTables` and `importTables` lists. I prefer the later since all the checks etc since there are kind of final lists after all other flags processing etc.

Reply:
I have split the filtering into two separate areas considering the resumability of import data, as well as the possibility of running start clean on an interrupted/fully completed migration. Please review the logic I have used considering all these cases

Reply:
Okay so you are filter here mainly, and other place is after the startClean because there you need to consider the tables cleaned. Is that correct?

Reply:
Yes, this is correct